src/Compilers/CSharp/Portable/Symbols/TupleTypeSymbol.cs

```diff
@@ -303,6 +308,158 @@ public override bool IsSealed
             }
         }
 
+        public override int Arity
+        {
+            get
+            {
+                return _underlyingType.Arity;
```

Comment:
Did not give a lot of thought to generics yet. I agree, modeling tuple as nongeneric type makes sense